layers/+source-control/git/README.org

```diff
@@ -59,6 +64,20 @@ file.
 Of course if your OS does not ship with git (!) you'll have to install it
 on your machine. You can download it from the [[http://git-scm.com/downloads][download page]].
 
+*** Basic Git configuration
+Define your git identity using the following commands in a terminal window
+replacing username and name@domain.tld with your own values
+
+#+begin_src shell
+  git config --global user.name "username"
+  git config --global user.email "name@domain.tld"
+#+end_src
+
+*** Using SSH URLs for remote repositories
+When using an SSH URL for remote repository access, [[https://help.github.com/articles/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent/][generate an SSH key]] and
+add it to your GitHub // GitLab account to remove the need to provide
```

Comment:
If you are explicitly mentioning GitHub/GitLab, it'll be better to add links to their documentations on adding SSH keys to the service.

[GitHub docs](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/adding-a-new-ssh-key-to-your-github-account)
[GitLab docs](https://docs.gitlab.com/ee/ssh/#add-an-ssh-key-to-your-gitlab-account)

Reply:
Yes, updated now.